mobile/lib/modules/album/views/album_viewer_page.dart

```diff
@@ -38,7 +38,7 @@ class AlbumViewerPage extends HookConsumerWidget {
 
     void selectionListener(bool active, Set<Asset> selected) {
       selection.value = selected;
-      multiSelectEnabled.value = true;
+      multiSelectEnabled.value = selected.isNotEmpty;
```

Comment:
In particular, this fixes this weird state:
![image](https://github.com/immich-app/immich/assets/100457/740a6130-95e3-44cf-833e-b4e963a2803f)